tx-pool/src/pool.rs

```diff
@@ -321,12 +214,20 @@ impl TxPool {
     // Remove transactions from the pool until total size < size_limit.
```

Comment:
```rust
    // Remove transactions from the pool until its size <= size_limit.
    tx_pool.limit_size(callbacks);
```
Code comments are not consistency. (I think it should be `<=` instead of `<`)